test/spec/modules/tpmnBidAdapter_spec.js

```diff
@@ -238,35 +241,107 @@ describe('tpmnAdapterTests', function () {
         expect(spec.isBidRequestValid(bid)).to.equal(false);
       });
 
-      it('should use bidder video params if they are set', () => {
-        const videoBidWithParams = utils.deepClone(VIDEO_BID);
-        const bidderVideoParams = {
-          api: [1, 2],
-          mimes: ['video/mp4', 'video/x-flv'],
-          playbackmethod: [3, 4],
-          protocols: [5, 6],
+      // it('when mediaType is Video - check', () => {
+      //   const bid = utils.deepClone(VIDEO_BID);
+      //   const check = {
+      //     w: 1024,
+      //     h: 768,
+      //     mimes: ['video/mp4'],
+      //     playbackmethod: [2, 4, 6],
+      //     api: [1, 2, 4, 6],
+      //     protocols: [3, 4, 7, 8, 10],
+      //     placement: 1,
+      //     minduration: 0,
+      //     maxduration: 60,
+      //     startdelay: 0,
+      //     plcmt: 1
+      //   };
+
+      //   expect(spec.isBidRequestValid(bid)).to.equal(true);
+      //   const requests = spec.buildRequests([bid], BIDDER_REQUEST);
+      //   expect(requests).to.deep.equal(null);
+
+      //   const request = requests[0].data;
+
+      //   expect(request.imp[0].video).to.deep.include({ ...check
+
+      //   });
+      // });
+
+      it('when mediaType New Video', () => {
```

Comment:
@dgirardi 


I have a question.

When I run the `gulp serve-and-test --file test/spec/modules/tpmnBidAdapter_spec.js` command locally, everything works fine.

However, when I run the `gulp test --file test/spec/modules/tpmnBidAdapter_spec.js` command locally, the test fails. 
The issue is that the `const requests = spec.buildRequests([NEW_VIDEO_BID], BIDDER_REQUEST);` code fails because the 'requests.imp.video' object is not being created.

I've checked multiple times, but I can't figure out what I missed.

For banners, the tests pass successfully.

Reply:
I'm reposting the code as follows after making sure it passes the test, taking into account using setTimeout in other test cases.

`
setTimeout(() => {
          expect(spec.isBidRequestValid(bid)).to.equal(true);
          const requests = spec.buildRequests([bid], BIDDER_REQUEST);
          const request = requests[0].data;
          expect(request.imp[0].video).to.deep.include({...check});
          done();
        }, 200);
`

Reply:
Through many trials and errors, 
I've come to realize that my test code didn't consider 'test-all-features-disabled' for the video type. 
I've made modifications to the test code and re-push it.